src/RecoveryServices/RecoveryServices/ChangeLog.md

```diff
@@ -20,6 +20,7 @@
 ## Upcoming Release
 * Azure Site Recovery support for removing a replicated disk.
```

Comment:
This will need to pull in the changes from the current release - which will be merged here tomorrow

Reply:
Okay.